### PR TITLE
celluloid: 0.16 → 0.17

### PR DESCRIPTION
**Pull request by jtojnar**
Renamed from gnome-mpv.

https://github.com/celluloid-player/celluloid/releases/tag/v0.17

- [x] Built on NixOS (sandboxed).
- [x] Tested playing a video.

### CODE REVIEW

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -111,6 +111,7 @@ mapAliases ({
   gettextWithExpat = gettext; # 2016-02-19
   git-hub = gitAndTools.git-hub; # added 2016-04-29
   glib_networking = glib-networking; # added 2018-02-25
+  gnome-mpv = celluloid; # added 2019-10-22
```
Comment (jonringer) [inline pkgs/top-level/aliases.nix]:
```suggestion
  gnome-mpv = celluloid; # added 2019-08-22
```

File: pkgs/applications/video/celluloid/default.nix [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    python3
+    appstream-glib
+    gettext
+    pkgconfig
+    desktop-file-utils
+    wrapGAppsHook
+  ];
+
+  buildInputs = [
+    epoxy
```
Comment (jonringer) [inline pkgs/applications/video/celluloid/default.nix]: get's rid of a warning, but I still can't see icons.
```suggestion
    epoxy
    hicolor-icon-theme
```

**Review by jonringer — APPROVED** [review]
`nix-review` passes on NixOS
diff LGTM (minor suggestions)
binary works
leaf package

File: pkgs/applications/video/celluloid/default.nix (hunk 2 above)
Comment (jtojnar) [reply pkgs/applications/video/celluloid/default.nix]: See the discussion here: https://github.com/NixOS/nixpkgs/pull/43150#issuecomment-450056204

Comment (jonringer) [reply pkgs/applications/video/celluloid/default.nix]: ah, interesting, guess i should install some icons then :smile:

**Pull request merged**